tests/uefistored/test_auth_var.py

```diff
@@ -62,3 +65,53 @@ def test_auth_variable(imported_vm):
 
     # Set the variable with new data, signed by a different cert
     set_and_assert_var(vm, Certificate(), b'this should fail', should_pass=False)
+
+    vm.shutdown(verify=True)
+
+
+def test_db_append(linux_uefi_vm):
+    """Pass if appending the DB succeeds. Otherwise, fail."""
```

Comment:
This rule is not enforced by `pydocstyle`?
Should it?

Reply:
Why change [PEP257](https://www.python.org/dev/peps/pep-0257/#one-line-docstrings)?

For one-line docstrings, it states:

> The docstring is a phrase ending in a period.

Which is why...

```python
def foo():
    """ Do nothing. """
```

Results in:

```bash
$ pydocstyle /tmp/foo.py

/tmp/foo.py:4 in public function `foo`:
        D210: No whitespaces allowed surrounding docstring text
```

Reply:
Please rebase your branch on master, this will add new checks regarding code & doc style.

Reply:
The current convention in our repositories is with whitespaces (I can find exceptions, such as VM.set_efi_var and VM.file_exists). I don't think we realized PEP257 advised otherwise.

If we want to change that, we'll have to do it globally (in a separate PR), and enforce it in our pydocstyle settings. Regardless of my own taste, it's probably better indeed.